.github/actions/deploy-environment/action.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy Environment
+
+inputs:
+  azure_credentials:
+    description: The GitHub access token
+    required: true
+  environment_name:
+    description: Name of the environment in Make file
+    required: true
+  root_folder:
+    description: The folder that contains the terraform to be validated
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+  - run: |
+      TERRAFORM_VERSION=$(awk '/{/{f=/^terraform/;next}f' terraform/terraform.tf | grep -o [0-9\.]*)
+      . config/platform-test.sh
+
+      if [ -z "$RESOURCE_GROUP_NAME" ]; then
+        echo "::error ::Failed to extract app_resource_group_name from $TFVARS"
```

Comment:
from $TFVARS -> from the confirguration?

Reply:
Copy paste error 😳 fixed the error message